slash/react/src/Tag/__tests__/Tag.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from "@testing-library/react";
+import { describe, expect, it } from "vitest";
+import { ComponentProps } from "react";
+import { Tag } from "../Tag";
+
+const classModifiers: ComponentProps<typeof Tag>["classModifier"][] = [
+  "success",
+  "info",
+  "warning",
+  "error",
+  "axa",
+  "black",
+  "purple",
+  "gray",
+  "danger",
+  "custom-class",
+];
+
+describe("Tag component", () => {
+  classModifiers.forEach((modifier) => {
+    it(`should render ${modifier} correctly with af-tag classname`, () => {
```

Comment:
Avec ça ce serait plus simple non ?
```suggestion
    it.each(classModifiers)(`should render "%s" correctly with af-tag classname`, (modifier) => {
```

Reply:
Good